src/plugins/security_oss/public/plugin.ts

```diff
@@ -51,8 +59,20 @@ export class SecurityOssPlugin
   }
 
   public start(core: CoreStart) {
+    const appState = this.appStateService.start({ core });
     return {
-      insecureCluster: this.insecureClusterService.start({ core }),
+      insecureCluster: this.insecureClusterService.start({ core, appState }),
+      anonymousAccess: {
+        async getAccessURLParameters() {
+          const { anonymousAccess } = await appState.getState();
+          return anonymousAccess.accessURLParameters;
+        },
+        async getCapabilities() {
+          return await core.http.get<Capabilities>(
```

Comment:
I think there was a verbal agreement in a Platform team long ago to do this since we don't define return type explicitly here and it's not always obvious that function returns `Promise`. I couldn't find it recorded anywhere, so will remove it.